packages/ui/src/components/_utils/analytics.ts

```diff
@@ -16,6 +16,7 @@ type ComponentSpecificClickEventProps = {
     Banner: { title: string }
     Button: { value: string, type: string }
     'Typography.Link': { value: string, href?: string }
+    Dropdown: { optionValue: string, optionKey?: string, optionKeyPath?: Array<string>, triggerValue?: string }
```

Comment:
`keyPath` in used in case of nested menus?

Reply:
Yes, it is for such cases. It seems we don't have them, but you never know